### ISSUE TITLE
Support module-level options in custom modules' Widget

### ISSUE THREAD

**Issue by JustSimplyKyle** [JakeStanger/ironbar#74]
**Is your feature request related to a problem? Please describe.**
Kinda? I want on_scroll_up and on_scroll_down in custom but that doesn't seem to work according to my testing

**Describe the solution you'd like**
Support for https://github.com/JakeStanger/ironbar/wiki/configuration-guide#32-module-level-options ! Should be quite simple I think.

**Additional context**
Why is it `on_click` instead of `on_click_left` in custom, very confusing

**Comment by JakeStanger** [comment]
This would be good for consistency, happy to add this in.

> Should be quite simple I think.

Probably not - the two are independent systems that work differently, hence why they have a different set of options from each other currently. The ideal way to implement this would be to make them share the same options/responsible code but that will require some refactoring.

> Why is it on_click instead of on_click_left in custom, very confusing

This got missed when I changed the module-level option's name for consistency. I'll get that fixed, thanks for spotting.

**Comment by JustSimplyKyle** [comment]
Thanks! Although it seems like on_scroll_up and on_scroll_down in the slider widget doesn't work
```corn
    $volume = {
        type = "custom"
        bar = [ 
            {
                type = "slider"
                class = "volbar"
                length = 100
                max = 100
                on_change = "!wpctl set-volume @DEFAULT_SINK@ $0%"
                on_scroll_down = "wpctl set-volume @DEFAULT_SINK@ 2%-"
                on_scroll_up = "wpctl set-volume @DEFAULT_SINK@ 2%+"
                on_click_right = "pavucontrol"
                value = "0:/home/kyle/.config/ironbar/scripts/volume"
                show_if = $check_sound_state
            }
        ]
    }
```

**Comment by JakeStanger** [comment]
For some reason the scale automatically binds to the scroll event, then does nothing with it, so the parent EventBox never receives the event.

I'm going to implement proper scrolling logic into the scale component, so it will just work to change the value on scroll without any config. There'll be a `step` config option to go with it to configure how much it steps by.

**Comment by JakeStanger** [comment]
*This* is what was resolved by #119, not #115 as it was linked.
